Added support for parsing a Pdf Buffer

simdrouin opened the pull request:
So that the library allows parsing a PDF file that is already as a buffer in memory or does not exists as a physical file

adrienjoly commented:
Hi @simdrouin, thanks for your contribution!
In order to validate and document this feature before merging it, would you mind including an automated test for it?

simdrouin commented:
Hi @adrienjoly, thank you for considering my pull request.
I have added a new file called parseAsBuffer.js and updated the README file accordingly.

Hope this helps!

adrienjoly commented:
Awesome! I will test this and hopefully merge your PR this week! :-)

Thanks for your contribution, @simdrouin ! 🙌
